runbotics/runbotics-orchestrator-ui/src/src-landing/components/BlogHeadImage/BlogHeadImage.module.scss

```diff
@@ -2,45 +2,48 @@
 
 
 
-
 .image {
-
     position: relative;
-    margin-top: $HEADER_HEIGHT;
+
     width: 100%;
     height: $BACKGROUND-HEIGHT;
-    background: url('/images/banners/blog-header.png') no-repeat;
+
+    margin-top: $HEADER_HEIGHT;
+
     background-position: center;
     background-size: cover;
     isolation: isolate;
     font-size: 250px;
     &::before {
-        content: '';
         position: absolute;
+
         width: 100%;
         height: 100%;
+
         z-index: $header-z-index;
-        background: transparent url('/images/blog/mask.svg');
+        background: #FBB040;
         mix-blend-mode: color;
-        opacity: 1;
+        content: '';
     }
     
     &::after{
         @include font-face-base;
         position: absolute;
-        z-index: 1;
-        content: "RunNews";
+
         bottom: -40px;
         right: 70px;
 
+        z-index: 1;

```

Comment:
add [isolation: isolate](https://developer.mozilla.org/en-US/docs/Web/CSS/isolation) on parent to prevent unwanted behaviour due to zindex